### PR TITLE
Update OSIRIS docs

### PR DESCRIPTION
**Pull request by MohamedAlmaki**
**Description of work.**
This a follow-up PR to #35362 that updates the documentation of the OSIRIS script, adds the release note, and config files example. In addition, this PR fixes a small bug in the empty subtraction scaling. 

**To test:**
- For the docs, build `docs-html` target then check the HTML files.
- For the empty subtraction, try to set the sample_empty_scale in the OSIRIS script to zero.

Fixes #35377. 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/techniques/ISISPowder-OSIRIS-v1.rst [hunk 1]
```diff
@@ -47,20 +47,39 @@ Methods
...
+  osiris_example.create_vanadium(run_number="119977", subtract_empty_can=False)
+
+.. _focus_osiris_isis-powder-diffraction-ref:
+
+focus
 ^^^^^^^^^^^^^^^^^^^^^^^^
-The *diffraction_focusing* method allows a user to process a series of runs into a
+The *focus* method allows a user to process a series of runs into a
 focused dSpace workspace. Whilst processing the runs the scripts can apply any corrections
 the user enables.
```
[the remark this answers is not in the document]
Comment (MohamedAlmaki) [reply docs/source/techniques/ISISPowder-OSIRIS-v1.rst:80]: ok

File: docs/source/techniques/ISISPowder-OSIRIS-v1.rst [hunk 2]
```diff
@@ -47,20 +47,45 @@ Methods
...
 
-On OSIRIS the following parameters are required when executing *create_vanadium*:
+The available corrections are:
+
+- empty container subtraction which can be enabled using :ref:`subtract_empty_can_osiris_isis-powder-diffraction-ref` parameter.
```
Comment (AnthonyLim23) [inline docs/source/techniques/ISISPowder-OSIRIS-v1.rst:84]: I am being fussy but "which" should have a comma before it., alternatively you can use "that" and not have a comma. Same for the line below. 

```suggestion
- empty container subtraction, which can be enabled using :ref:`subtract_empty_can_osiris_isis-powder-diffraction-ref` parameter.
```

Comment (MohamedAlmaki) [reply docs/source/techniques/ISISPowder-OSIRIS-v1.rst:84]: Okay

**Review by AnthonyLim23 — APPROVED** [review]
Docs look good. The script behaves as expected, if I do 0 for the scale factor (blue line) and 1 for the scale (orange line), then the orange is below the blue (data was subtracted). 

image

**Comment by AnthonyLim23** [comment]
```

from isis_powder.osiris import Osiris
from isis_powder.pearl import Pearl

osiris_focus = Osiris(user_name="Calib",
                  output_directory=r"C:\Users\BTR75544\Downloads\OsirisCalibrationFiles (1)\OsirisCalibrationFiles\output",
                  config_file=r"C:\Users\BTR75544\Downloads\OsirisCalibrationFiles (1)\OsirisCalibrationFiles\Config.yaml",
                  calibration_directory=r"C:\Users\BTR75544\Downloads\OsirisCalibrationFiles (1)\OsirisCalibrationFiles\Calibration_files",
                  calibration_mapping_file=r"C:\Users\BTR75544\Downloads\OsirisCalibrationFiles (1)\OsirisCalibrationFiles\Calibration_files\osiris_run_mapping.yaml")
                  

osiris_focus.create_vanadium(run_number="119977,119978")

osiris_focus.focus(run_number="119977,119978",
    merge_drange=True,
    vanadium_normalisation=True,
    subtract_empty_can=True,
    sample_empty_scale=0.1
)
```

**Pull request merged**